sw-ui/src/main/kotlin/org/luxons/sevenwonders/ui/components/game/BoardSummary.kt

```diff
@@ -47,10 +53,11 @@ private fun createFullBoardPreview(board: Board): ReactElement = buildElement {
     }
 }
 
-private fun RBuilder.boardSummary(
+fun RBuilder.boardSummary(
     player: PlayerDTO,
     board: Board,
     side: BoardSummarySide,
+    showPreparationStatus: Boolean = true,
```

Comment:
I'm wondering whether we should add an enum for this: `PreparationStatusPosition` with values TOP_BAR, BOTTOM_BAR, NONE